ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -157,6 +176,85 @@ export const ProfilePage = withUserProfile()(class extends React.Component<
       </div>;
     };
 
+    function renderInstitutionalAffiliationComponents(componentHandle: React.Component) {
+      if (requireInstitutionalVerification) {
```

Comment:
The logic here could probably use a bit of cleanup. Once we're no longer rendering everything in one big TSX template, I usually tend to favor explicit control flow. You could also defer evaluating this.profile into the helper method, so this method becomes simply:

const {requireInstitutionalVerification} = this.configStore.getValue();
if (requireInstitutionalVerification) {
  return this.renderVerifiedInstitutionalAffiliationComponents();
} else {
  return this.renderOldInstitutionalAffiliationComponents();
}